src/main/java/com/bean00/RequestParser.java

```diff
@@ -3,38 +3,35 @@
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 
 public class RequestParser {
     private BufferedReader in;
+    private HashMap<String, String> requestLineData = new HashMap<>();
```

Comment:
Originally it was so that I could collect multiple pieces of data (the requestMethod and requestURL), and return them in 1 data structure. However, I'm just taking that hashmap and getting the values directly out of it, so the hashmap isn't really necessary here. I'm using `String[]` and `String`s instead now